app.rb

```diff
@@ -0,0 +1,10 @@
+require 'sinatra/base'
+
+class TillTest < Sinatra::Base
+  get '/' do
+    'Hello TillTest!'
+  end
+
+  # start the server if ruby file executed directly
+  run! if app_file == $TillTest
```

Comment:
Style/GlobalVars: Do not introduce global variables. (https://github.com/bbatsov/ruby-style-guide#instance-vars, http://www.zenspider.com/Languages/Ruby/QuickRef.html)